ip/resolver.go

```diff
@@ -1,6 +1,6 @@
-package ipify
+package ip
 
-type Client interface {
+type Resolver interface {
 	GetPublicIP() (string, error)
```

Comment:
Could be 2 implementations with same `ip. Resolver` interface e.g. `NewPublicResolver()`, `NewOutboundResolver()`